src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -2128,6 +2129,10 @@ AnnotationCollector::annotation_index(const ClassLoaderData* loader_data,
       if (!privileged)              break;  // only allow in privileged code
       return _method_Scoped;
     }
+    case VM_SYMBOL_ENUM_NAME(jdk_internal_vm_annotation_NativeAccess_signature): {
```

Comment:
I don't think this stuff is needed anymore? The annotation should just be there for javadoc puprposes.

Reply:
Yes. pushed to remove most hotspot changes